lib/workload/stateless/stacks/filemanager/deploy/lib/filemanager.ts

```diff
@@ -101,5 +120,44 @@ export class Filemanager extends Stack {
       eventSources: [queue],
       buckets: props.eventSourceBuckets,
     });
+
+    let objectsQueryLambda = new ObjectsQueryFunction(this, 'ObjectsQueryFunction', {
+      vpc: vpc,
+      host: host,
+      port: props.port,
+      securityGroup: lambdaSecurityGroup
+    });
+
+    const ObjectsQueryFunctionIntegration = new HttpLambdaIntegration(
+      "FilemanagerQueryApiIntegration",
+      objectsQueryLambda.function
+    );
+
+    const pool = new UserPool(this, "userPool", {
+      userPoolName: "OrcaBusUserPool",
+    });
+
+    // JWT authorizer
+    const authorizer = new HttpJwtAuthorizer(
+        id + "FilemanagerQueryApiAuthorizer",
+        `https://cognito-idp.${this.region}.amazonaws.com/`,
+        {
+          identitySource: ["meow"], //["$request.header.Authorization"],
```

Comment:
meow meow

Reply:
Just wanted to `cdk synth`, so I threw the cat in there X"D